gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -538,5 +550,251 @@ def mask_rcnn_fpn_resnet101_v1d_coco(pretrained=False, pretrained_base=True, **k
         scales=(2, 4, 8, 16, 32), ratios=(0.5, 1, 2), alloc_size=(384, 384),
         rpn_nms_thresh=0.7, rpn_train_pre_nms=12000, rpn_train_post_nms=2000,
         rpn_test_pre_nms=6000, rpn_test_post_nms=1000, rpn_min_size=0,
-        num_sample=512, pos_iou_thresh=0.5, pos_ratio=0.25, deep_fcn=True,
+        num_sample=512, pos_iou_thresh=0.5, pos_ratio=0.25, target_roi_scale=2,
+        num_fcn_convs=4, **kwargs)
+
+
+def mask_rcnn_resnet18_v1b_coco(pretrained=False, pretrained_base=True, rpn_test_pre_nms=6000,
```

Comment:
@zhreshold any thoughts on this change of parameters? Should we do that for all rcnn-family models? Should we take a closer look at what other params should we align with `rpn_test_pre_nms ` and `rpn_test_post_nms`?

At least I see `rcnn_max_dets` should not be larger than `rpn_test_post_nms`. Maybe they should be the same.

Reply:
I think we should take a look at all these pre_nms and post_nms arguments, some are not effective in one mode vs another (with or w/o RPN). Some arguments have dependencies as @hetong007 just mentioned, we need more proper sanify checks for these. @Jerryzcn